settings.py

```diff
@@ -24,5 +24,9 @@
 USE_TZ = False
```

Comment:
Consider setting `USE_TZ = True` to enable timezone-aware datetimes, which is generally recommended for Django projects to handle timezones correctly.